### PR TITLE
Modify timetable

### PR DESCRIPTION
**Pull request by mchiapello-test**
Changed the program based on the classroom time availability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,22 +10,18 @@
 
 
    - Day1 
-	   - 9-11: 		Course introduction
-	   - 11-12:30: 		UNIX
-	   - 12:30-13:30: 	Lunch break
-	   - 13:30-18: 		UNIX
-
+	   - 8-10: 		Course introduction
```
Comment (mchiapello) [inline README.md:13]: Are we sure we can start so early?

File: README.md [hunk 2]
```diff
@@ -10,22 +10,18 @@
...
+	   - 10-14:00: 		UNIX
 
    -  Day2
```
Comment (mchiapello) [inline README.md:16]: Everything a little bit squeezed!

File: README.md (hunk 1 above)
Comment (mchiapello-test) [reply README.md:13]: Wait for Andrea's confirmation of classroom booking!!

File: README.md (hunk 2 above)
Comment (mchiapello-test) [reply README.md:16]: We can move something from day 2 to day 3! Do it on fly...

**Pull request closed without merging**